tests/unit/test_git.py

```diff
@@ -6,6 +6,12 @@
 url_for = create_url_helper('https://api.github.com/repos/octocat/Hello-World/'
                             'trees/9fb037999f264ba9a7fc6274d15fa3ae2ab98312')
 
+delete_url_for = create_url_helper('https://api.github.com/repos/octocat/Hello-World/'
```

Comment:
> reference_url_for?

Hey @sigmavirus24 I updated the variable name - hopefully that was the suggestion.  I also rebased so that the branch is up to date. Thanks for taking a look at this and let me know if there's anything else with this PR that needs to be fixed.